index.html

```diff
@@ -85,55 +101,80 @@ <h3 class="project_title">Profesional Art Printing Data</h3>
-         <li class="language"><img src="./images/Rectangle 55.svg">Languages</li>
-          <ul class="lang">
-            <li class="li_lang">Javascript</li>
-            <li class="li_lang">Ruby</li>
-            <li class="li_lang">HTML</li>
-            <li class="li_lang">CSS</li>
-          </ul>
-        <li class="frameworks"><img src="./images/Rectangle 56.svg">Frameworks</li>
-          <ul class="framewk">
-            <li class="li_framewk">React.js</li>
-            <li class="li_framewk">Ruby on Rails</li>
-            <li class="li_framewk">RSpec</li>
-            <li class="li_framewk">Capybara</li>
-          </ul>
-        <li class="skills"><img src="./images/Rectangle 57.svg">Skills</li>
-          <ul class="sk">
-            <li class="li_skills">Database Management</li>
-            <li class="li_skills">Version Control</li>
-            <li class="li_skills">CLI</li>
-            <li class="li_skills">API Design</li>
-            <li class="li_skills">Web Development</li>
-          </ul>
+  </section>
+  <section class="about_me">
+    <div class="about_me_intro">
+      <h1 class="about_me_title">About Me</h1>
+      <p class="description1">Hello I`m a software developer! I can help you build a product , feature or website Look
+        through some of my work and experience! If you like what you see and have a project you need coded, don’t
+        hesitate to contact me.</p>
+      <a href="#" class="resume">Get my resume</a>
+    </div>
+    <ul class="list">
+      <li class="language"><img src="./images/Rectangle 55.svg">Languages</li>
+      <ul class="lang">
+        <li class="li_lang">Javascript</li>
+        <li class="li_lang">Ruby</li>
+        <li class="li_lang">HTML</li>
+        <li class="li_lang">CSS</li>
+      </ul>
+      <li class="frameworks"><img src="./images/Rectangle 56.svg">Frameworks</li>
+      <ul class="framewk">
+        <li class="li_framewk">React.js</li>
+        <li class="li_framewk">Ruby on Rails</li>
+        <li class="li_framewk">RSpec</li>
+        <li class="li_framewk">Capybara</li>
+      </ul>
+      <li class="skills"><img src="./images/Rectangle 57.svg">Skills</li>
+      <ul class="sk">
+        <li class="li_skills">Database Management</li>
+        <li class="li_skills">Version Control</li>
+        <li class="li_skills">CLI</li>
+        <li class="li_skills">API Design</li>
+        <li class="li_skills">Web Development</li>
       </ul>
-    </section>  
-  </body>
+    </ul>
+  </section>
+  <section class="contact">
+    <p class="contact_description">I'm always interested in hearing about new projects, so if you'd like to chat please
+      get in touch.</p>
+    <form class="contact_form" action="https://formspree.io/f/xqkoreon" method="POST">
+      <input type="text" class="name" placeholder="Full name">
+      <input type="email" class="email" placeholder="Email address">
+      <input type="textarea" class="message" placeholder="Write me something...">
```

Comment:
- Kindly update this element to use a `<textarea>` element tag instead of an input tag, as the textarea element tag is used in forms for long text👍
- Also, please add HTML validation to this element to restrict the maximum number of characters entered here to 500 characters to adhere to the project requirements🙏